README.md

```diff
@@ -1,19 +1,23 @@
-# Math Magicians
-
 <a name="readme-top"></a>
 
+  <h3><b>Math magicians</b></h3>
```

Comment:
- [ ] You are yet to remove the CRA boilerplate files.

  In the `public` folder kindly remove these files:
  - logo192.png
  - logo512.png
  - robots.txt